### PR TITLE
Migration to package format 2

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

**Comment by fmessmer** [comment]
check message_runtime in CATKIN_DEPENDS

File: cob_pick_place_action/package.xml [hunk 1]
```diff
@@ -12,38 +12,26 @@
...
-  <build_depend>std_msgs</build_depend>
-  <build_depend>tf</build_depend>
-  
-  <run_depend>actionlib_msgs</run_depend>
-  <run_depend>actionlib</run_depend>
-  <run_depend>cob_grasp_generation</run_depend>
-  <run_depend>cob_moveit_interface</run_depend>
-  <run_depend>geometric_shapes</run_depend>
-  <run_depend>geometry_msgs</run_depend>
   <run_depend>message_runtime</run_depend>
```
Comment (NadiaHG) [inline cob_pick_place_action/package.xml]: I thought <  run_depend  > tag is no longer allowed.. isn't it?

**Comment by mgruhler** [comment]
Fine for me

File: cob_pick_place_action/package.xml (hunk 1 above)
Comment (fmessmer) [inline cob_pick_place_action/package.xml]: done

**Comment by NadiaHG** [comment]
LGTM

**Pull request merged**